mep/footnotes/urls.py

```diff
@@ -1,10 +1,12 @@
 from django.conf.urls import url
-from mep.footnotes.views import BibliographyAutocomplete
+
+from mep.footnotes.views import BibliographyAutocomplete, CardList
 
 # url namespace
 app_name = 'footnotes'
 
 urlpatterns = [
     url(r'^bibliography/autocomplete/$', BibliographyAutocomplete.as_view(),
         name='bibliography-autocomplete'),
+    url(r'^cards/$', CardList.as_view(), name='card-list'),
```

Comment:
not sure if this matters, but the other views and urls use plurals here, e.g. the view is `MembersList` and the route name is `members-list`

Reply:
Thanks for noting, I think I was inconsistent about that. For some reason cards-list sounds weird to me, but I agree we should be consistent. Simplest to switch to plural here?

Reply:
it sounds weird to me too! but yeah, i guess we should be consistent and switch to plural.